packages/app/src/theme/aperture.ts

```diff
@@ -95,7 +95,7 @@ const baseTheme = createTheme({
 });
 
 const createCustomThemeOverrides = (
-  theme: BackstageTheme,
+  theme: Theme,
```

Comment:
Would this help? Smells like the entry point to the problems to me. At least ad-hoc.

```suggestion
  theme: UnifiedTheme,
```

Reply:
Gave that a try but then there's type errors further on. For example we do `theme.spacing(1)` which gives us: "Property 'spacing' does not exist on type 'UnifiedTheme'."